qiskit/circuit/__init__.py

```diff
@@ -56,6 +56,48 @@
 Supplementary Information
 =========================
 
+.. dropdown:: Quantum Circuit with conditionals
+   :animate: fade-in-slide-down
+
+   When building a quantum circuit, there can be interest in applying a certain gate only
+   if a classical register has a specific value. This can be done with the
+   :meth:`InstructionSet.c_if` method.
+
+   In the following example, we start with a single-qubit circuit formed by only a Hadamard gate,
+   in which we expect to get :math:`|0\\rangle` and :math:`|1\\rangle` with equal probability.
+
+   .. jupyter-execute::
+
+      from qiskit import BasicAer, transpile
+
+      qc = QuantumCircuit(1, 1)
+      qc.h(0)
+      qc.measure(0, 0)
+      display(qc.draw())
+
+      backend = BasicAer.get_backend('qasm_simulator')
+      tqc = transpile(qc, backend)
+      counts = backend.run(tqc).result().get_counts()
+
+      print(counts)
```

Comment:
If we're going to execute this with a `jupyter-execute` I'd split this into 2 cells and avoid the `display()` call because that doesn't mean anything in a context outside of jupyter. Something like:

```suggestion
   .. jupyter-execute::

      from qiskit import BasicAer, transpile

      qc = QuantumCircuit(1, 1)
      qc.h(0)
      qc.measure(0, 0)
      qc.draw('mpl')
      
   .. jupyter-execute::
   
      backend = BasicAer.get_backend('qasm_simulator')
      tqc = transpile(qc, backend)
      counts = backend.run(tqc).result().get_counts()

      print(counts)
```

I'm not super convinced the `jupyter-execute` is necessary for this example because we can demonstrate this fairly easily without running code (minus the circuit diagram), but it doesn't hurt I guess.